hack/run-functests.sh

```diff
@@ -34,6 +32,9 @@ elif [ "$FEATURES" == "" ]; then
   exit 1
 else
   FOCUS="-ginkgo.focus="$(echo "$FEATURES" | tr ' ' '|')
+  if [ "$FOCUS_TESTS" != "" ]; then
```

Comment:
This new environment variable should be documented in `cnf-tests/README` as a replacement for "-ginkgo.focus"

Reply:
done